docs/pages/bssw/bssw_styling_originalarticles.md

```diff
@@ -70,6 +70,18 @@ This section has details of the author in a short paragraph. The content is free
 ## Metadata section
 There is no specific guidance for this for the event content type. See [common metadata section](bssw_content_metadata.html) of the guide.
 
+## Citations/References
+
+Certain content types on [bssw.io](https://bssw.io) do not require formal reference styling (e.g. formal *citations*) either because the format is highly constrained (e.g. Events, Curated Content) or because the document is short and uses only a small number of inlined hyperlinks. There are, however, circumstances where formal reference styling (e.g. formal *citations*) is warranted. These include...
+
+* Larger documents with *many* references that can benefit from formal reference styling to improve [readability and reading comprehension](https://www.psychologytoday.com/us/blog/your-childs-brain-and-behavior/201701/the-effects-digital-technology-reading?amp).
+
+* Documents that reference multiple off-line materials (e.g. books) for which there is no, suitable on-line (e.g. hyperlink-able) proxy.
+
+* Content that in the judgement of EB members and/or authors requires references to more fully support perhaps sensitive and/or controversial positions (e.g. the Covid19 article).
+
+The decision to *allow* or *require* references is one that should be agreed upon by the author and EB members prior to developing the content. When references are to be used, we require authors to use the less intrusive [reference links](https://www.markdownguide.org/basic-syntax#reference-style-links) ([full spec](https://github.github.com/gfm/#reference-link)) and to follow the guidelines described [here](https://github.com/betterscientificsoftware/bssw.io/blob/master/Articles/Blog/ReferencesInMarkdownHybridApproach.md) where the [`wikize_refs.py`](https://github.com/betterscientificsoftware/bssw.io/blob/master/utils/README.md#wikize_refspy) tool can be helpful.
+
 ## Existing Examples
 
 * A skeleton Markdown template for a blog article, which one can copy and customize is available at [this location](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/blob/master/Articles/Blog/BlogArticleSkeletonA.md)
```

Comment:
Can we include an example and link to it that includes use of references? Maybe `WhenNotToWriteAutomatedTests.md` here and indicate its a good example of the use of citations/references.

Reply:
@markcmiller86, just be clear, should the link to `WhenNotToWriteAutomatedTests.md` as an example go into the section `Existing Examples` or the section on citations/references?

Reply:
I think `Existing Examples` as I believe that to be a list of examples that demonstrate various useful aspects what is described here. But, each example alone is probably not sufficient and maybe a short line or phrase that goes with the example(s) as to *what* the example is good for demonstrating (thought I wouldn't expect you to do that for all the pre-existing examples there, at least do it for the new one you will add that demos citrations). Does that make sense?

Reply:
@markcmiller86, the link to the example file is in commit https://github.com/betterscientificsoftware/bssw.io/pull/750/commits/f1970b0be8264c8e89cf2ac584f0d4ff2ddc0fc9.  

NOTE: This link points to this `*.md` file on the 'master' branch but the updated `*.md` file is on this PR branch so that link will only point to the updated example once this PR gets merged.